README.md

```diff
@@ -19,22 +19,22 @@ Using Maven:
 See more examples in the `./samples` folder.
 
 ```java
-import com.coveo.pushapiclient.DocumentBuilder;
-import com.coveo.pushapiclient.Source;
+import com.coveo.document.DocumentBuilder;
+import com.coveo.source.SourceClient;
```

Comment:
If these paths are changing, does that mean we are introducing a breaking change for the users of the sdk?

Reply:
you are right i didnt pay attention to that ,  i guess i can't play with any classes public to the client